optimum/exporters/openvino/convert.py

```diff
@@ -546,8 +540,12 @@ def export_from_model(
             f"Compression of the weights to {ov_config.quantization_config} requires nncf, please install it with `pip install nncf`"
         )
 
-    library_name = TasksManager._infer_library_from_model_or_model_class(model=model)
-    TasksManager.standardize_model_attributes(model)
+    library_name = None
+    if "zero-shot-image-classification" in task and 'clip' in getattr(model.config, 'model_type'):
```

Comment:
can we have more robust way to understand that it is open_clip? TaskManager checks available files for that, can existence open_clip_config.json be signal that we have open_clip model?

Reply:
done with ov_infer_library_from_model